src/CMake/findpackage.cmake

```diff
@@ -12,13 +12,13 @@ string(TOLOWER ${PROJECT_NAME} LOWER_NAME)
 # For use by xrt consumers (using cmake) to import xrt libraries
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`